<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: #EEE;
	background-color: qlineargradient(spread:pad, x1:0.119, y1:0.647409, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(37, 37, 37, 255));
}

#title {
	background-color: transparent;
	font-size: 26px;
	color: rgba(13, 83, 170, 0.9);
}

#error {
	background-color: transparent;
	color: rgba(255, 0, 0, 0.8);
}

QLabel {
	font-size: 20px;
	background: transparent;
}

QLineEdit {
	border-radius: 10px;
	background-color: rgba(20, 20, 20, 0.75);
	padding: 5px 10px;
	border:  1px solid rgba(255, 255, 255, 0.25);
	border-bottom-color: rgba(110, 110, 255, 0.25);
	font-size: 16px;
}
QLineEdit:focus {
	border-bottom-color: #0d53aa;
}

QPushButton {
	border-radius: 10px;
	padding: 5px 10px;
	border: 1px solid #0d53aa ;
	background-color: #0d53aa; 
	font-weight: 900;
	font-size: 16px;
}
QPushButton:hover {
	background-color: red; 
	border-color: red;
}

#cancel_btn {
	border-color: rgba(84, 84, 84, 0.75);
	background-color: rgba(84, 84, 84, 0.75);
}</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DOWNLOAD youtube videos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Video URL :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Resolution :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download_btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vid_url">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ex : https://www.youtube.com/watch?v=BaW_jenozKc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vid_title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Optional</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vid_res">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Optional</string>
   </property>
  </widget>
  <widget class="QLabel" name="res">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(13, 83, 170, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>vid_url</tabstop>
  <tabstop>vid_title</tabstop>
  <tabstop>vid_res</tabstop>
  <tabstop>download_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
